src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -365,17 +365,21 @@ class AccessibleView extends Disposable {
 		const previousProvider = Object.assign({}, this._currentProvider);
 		const accessibleViewHelpProvider = Object.assign({}, this._currentProvider);
 		accessibleViewHelpProvider.options.type = AccessibleViewType.Help;
-		accessibleViewHelpProvider.provideContent = () => this._getAccessibleViewHelpDialogContent(accessibleViewHelpProvider!);
+		accessibleViewHelpProvider.provideContent = () => this._getAccessibleViewHelpDialogContent(accessibleViewHelpProvider);
 		accessibleViewHelpProvider.onClose = () => this.show(previousProvider);
-		this._accessiblityHelpIsShown.set(true);
-		const delegate: IContextViewDelegate = {
-			getAnchor: () => { return { x: (window.innerWidth / 2) - ((Math.min(this._layoutService.dimension.width * 0.62 /* golden cut */, DIMENSIONS.MAX_WIDTH)) / 2), y: this._layoutService.offset.quickPickTop }; },
-			render: (container) => {
-				container.classList.add('accessible-view-container');
-				return this._render(accessibleViewHelpProvider, container, true);
-			},
-		};
-		this._contextViewService.showContextView(delegate);
+		accessibleViewHelpProvider.options.ariaLabel = localize('accessibleViewHelp', "Accessible View Help");
+		this._contextViewService.hideContextView();
+		setTimeout(() => {
+			this._accessiblityHelpIsShown.set(true);
+			const delegate: IContextViewDelegate = {
+				getAnchor: () => { return { x: (window.innerWidth / 2) - ((Math.min(this._layoutService.dimension.width * 0.62 /* golden cut */, DIMENSIONS.MAX_WIDTH)) / 2), y: this._layoutService.offset.quickPickTop }; },
+				render: (container) => {
+					container.classList.add('accessible-view-container');
+					return this._render(accessibleViewHelpProvider, container, true);
+				},
+			};
+			this._contextViewService.showContextView(delegate);
+		}, 100);
```

Comment:
Can 0 (or no time) work here? That will put the execution on the next task, if that doesn't work then you can't guarantee this won't introduce a race condition.

Reply:
0 does not work here. based on my testing, 50 ms works fine for this and other such instances.

Reply:
it's not ideal, but I'm not sure of a better solution after spending a lot of time trying to come up with one

Reply:
Do you have a link to what this is fixing? We don't want to just add a hack that we need to fix later, is there some event we could listen to other than an arbitrary time?

Reply:
fixing the same issue as #186514

the focused element changes, but the screen reader (voice over) does not detect that focus change if we don't wait a little bit

Reply:
Does hiding the context view remove the element or just hide it?

Reply:
Just hides it. Maybe I should try disposing of it?

Reply:
tried removing the element and I don't think that's what we want to do here

Reply:
If the context menu doesn't remove it, that might be the cause of the issue?

Reply:
that's how the context view service works and I don't see other things removing the element

Reply:
Maybe the combination of removing and `setTimeout(, 0)` will do it? Shame if not, but let's add a `// HACK: desc...` comment to make it clear what's happening